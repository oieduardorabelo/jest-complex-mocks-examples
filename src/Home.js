import React from "react";

function Home(props) {
  return <div><p>Home component from Route! {JSON.stringify(props)}</p></div>;
}

export default Home;
