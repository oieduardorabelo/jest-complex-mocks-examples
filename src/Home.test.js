import React from "react";
import { render, mount } from "enzyme";
import toJson from "enzyme-to-json";

import Home from "./Home";

it("renders correctly", () => {
  const actual = toJson(render(<Home />));
  expect(actual).toMatchSnapshot();
});
