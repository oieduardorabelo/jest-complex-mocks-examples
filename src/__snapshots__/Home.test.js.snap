// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Home component from Route! {}
  </p>
</div>
`;
